integration/integration_test.sh

```diff
@@ -2,7 +2,7 @@
 INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
```

Comment:
Yeah, I worried that might affect some of the subscripts but they all do `set -e` anyhow so Done.